[32minfo[39m: Pass an object and this works {"additional":"properties","are":"passed along","service":"","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: Use a helper method if you want {"additional":"properties","are":"passed along","service":"","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: Pass a message and this works {"service":"","additional":"properties","are":"passed along","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: Use a helper method if you want {"service":"","additional":"properties","are":"passed along","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: test message my string {"service":"","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: test message 123 {"service":"","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: test message first, second {"service":"","timestamp":"2019-07-11 13:53:59","number":123}
[32minfo[39m: Found error at Thu Jul 11 2019 13:53:59 GMT+0800 (GMT+08:00) {"service":"","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: Found error at Error: chill winston {"service":"","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: Found error at /WUT/ {"service":"","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: Found error at true {"service":"","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: Found error at 100 {"service":"","timestamp":"2019-07-11 13:53:59"}
[32minfo[39m: Found error at 1, 2, 3 {"service":"","timestamp":"2019-07-11 13:53:59"}
[33mwarn[39m: Error passed as info {"service":"","timestamp":"2019-07-11 13:53:59","stack":"Error: Error passed as info\n    at Object.<anonymous> (E:\\workspace\\branch\\node-market\\bin\\log\\index.js:101:13)\n    at Module._compile (internal/modules/cjs/loader.js:776:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)\n    at Module.load (internal/modules/cjs/loader.js:653:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:585:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)\n    at startup (internal/bootstrap/node.js:283:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)"}
[31merror[39m: Error passed as message {"service":"","timestamp":"2019-07-11 13:53:59","stack":"Error: Error passed as message\n    at Object.<anonymous> (E:\\workspace\\branch\\node-market\\bin\\log\\index.js:102:21)\n    at Module._compile (internal/modules/cjs/loader.js:776:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)\n    at Module.load (internal/modules/cjs/loader.js:653:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:585:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)\n    at startup (internal/bootstrap/node.js:283:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)"}
[33mwarn[39m: Maybe important error: Error passed as meta {"service":"","stack":"Error: Error passed as meta\n    at Object.<anonymous> (E:\\workspace\\branch\\node-market\\bin\\log\\index.js:104:40)\n    at Module._compile (internal/modules/cjs/loader.js:776:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)\n    at Module.load (internal/modules/cjs/loader.js:653:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:585:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)\n    at startup (internal/bootstrap/node.js:283:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)","timestamp":"2019-07-11 13:53:59"}
[31merror[39m: Important error: Error passed as meta {"service":"","stack":"Error: Error passed as meta\n    at Object.<anonymous> (E:\\workspace\\branch\\node-market\\bin\\log\\index.js:105:42)\n    at Module._compile (internal/modules/cjs/loader.js:776:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)\n    at Module.load (internal/modules/cjs/loader.js:653:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:585:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)\n    at startup (internal/bootstrap/node.js:283:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)","timestamp":"2019-07-11 13:53:59"}
[31merror[39m: Error as info {"service":"","timestamp":"2019-07-11 13:53:59","stack":"Error: Error as info\n    at Object.<anonymous> (E:\\workspace\\branch\\node-market\\bin\\log\\index.js:107:14)\n    at Module._compile (internal/modules/cjs/loader.js:776:30)\n    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)\n    at Module.load (internal/modules/cjs/loader.js:653:32)\n    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)\n    at Function.Module._load (internal/modules/cjs/loader.js:585:3)\n    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)\n    at startup (internal/bootstrap/node.js:283:19)\n    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)"}
